routes/web.php

```diff
@@ -22,8 +22,6 @@
 
 Route::view('kiosk', 'kiosk')->name('kiosk');
 
-Route::redirect('privacy', 'https://www.gatech.edu/privacy');
-Route::redirect('nova/logout', 'logout');
```

Comment:
Is this really safe to remove?

Reply:
Seems like it. The logout link in the drop-down already goes to `/logout`

Reply:
Sounds good then.